test/com/yetanalytics/lrs_admin_ui/functions/time_test.cljs

```diff
@@ -0,0 +1,20 @@
+(ns com.yetanalytics.lrs-admin-ui.functions.time-test
+  "Test time utility functions. Note that any headless browser running this
+  should have TZ=America/New_York set."
+  (:require [clojure.test :refer-macros [deftest is testing]]
+            [com.yetanalytics.lrs-admin-ui.functions.time
+             :refer [tz-offset-mins*
+                     utc->local-datetime]]))
+
+(deftest tz-offset-mins*-test
+  (testing "yields different offsets for different dates"
```

Comment:
Might want to add a comment saying that it's because of Daylight Savings Time (I know if I come back to this months later I'd get confused).

Reply:
I'm gonna put it in the `testing` string

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/109/commits/3eaec9fdf81235f760b695af784af2caf6e399fd